pkg/kconfig/kconfig.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kconfig
+
+import (
+	"bufio"
+	"compress/gzip"
+	"fmt"
+	"os"
+	"strings"
+)
+
+const (
+	containerCgroupPath = "/proc/1/cgroup"
+)
+
+var ebpfCheckOptions = []string{
+	"CONFIG_BPF",
+	"CONFIG_BPF_SYSCALL",
+	"CONFIG_HAVE_EBPF_JIT",
+	"CONFIG_BPF_JIT",
+	"CONFIG_BPF_JIT_ALWAYS_ON",
```

Comment:
I don't think the BPF JIT is necessary, right? cc @Sylfrena

Reply:
Yeah not necessary but I think it'll be quite slow otherwise: https://elixir.bootlin.com/linux/latest/source/kernel/bpf/Kconfig#L40 So I personally think we should keep it as a mandatory check. But we can probably see how many users we come across where it doesn't enabled by default and can move it to 'nice to have' then?

Reply:
Sounds good